compiler/codegen/OMRCodeGenPhase.cpp

```diff
@@ -449,13 +489,15 @@ OMR::CodeGenPhase::performSetupForInstructionSelectionPhase(TR::CodeGenerator *
                appendTreeTop = currentTreeTopToappendTreeTop[appendTreeTop];
                }
 
-            // Anchor the l2a before the current treetop
+            // Anchor the aloadi before the current treetop
             appendTreeTop->insertBefore(anchorTreeTop);
             currentTreeTopToappendTreeTop[iter.currentTree()] = anchorTreeTop;
 
             traceMsg(comp, "GuardedStorage: Anchored  %p to treetop = %p\n", node, anchorTreeTop);
             }
          }
+
+      comp->getBitVectorPool().release(unAnchorableAloadiNodes);
```

Comment:
Not relevant to this PR but this seems like a great candidate for RAII. @mgaudet any thoughts?